### DISCUSSION TITLE
How to register custom Liquid tag

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#9859] [Q&A]
According to new implementation all the Liquid tags are restistered within https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewParser.cs

I see there is a lambda https://github.com/OrchardCMS/OrchardCore/blob/c75d5fcaa60977f51cddc0632c0d7f5265b8aa5d/src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewOptions.cs#L21 that allows to apply some configuration.

But in order to register a custom tag I need to call something like `RegisterParserTag("captcha", ArgumentsList, ReCaptchaTag.WriteToAsync);` and `ArgumentsList` is a protected property.
So how can I achieve that?

@sebastienros

**Comment by ns8482e** [comment]
In startup use `services.Configure<LiquidViewOptions>(o =>  )`
